lib/csv_helper/loader.rb

```diff
@@ -19,9 +19,23 @@ def load(file, options = {})
     def load_records(file, options)
       records = []
 
-      records = klass == Institution ? load_csv_with_version(file, records, options) : load_csv(file, records, options)
+      records = case name
+                when Program.name
+                  load_csv_with_row(file, records, options)
+                when Institution.name
+                  load_csv_with_version(file, records, options)
+                else
+                  load_csv(file, records, options)
+                end
+
       results = klass.import records, ignore: true, batch_size: Settings.active_record.batch_size.import
+
+      results.failed_instances.each do |record|
+        record.errors.add(:row, record.try(:csv_row))
+      end
```

Comment:
Why the move?  I think it should be moved back.
* This only works for `Program` objects
* Every other import has the row populated by at `after_import_validations`, line 97

Reply:
Moved this to only occur when making a call to validator's `after_import_batch_validations` method, this keeps the call generic and only runs it when needed

Reply:
Yeah, this makes sense to me assuming more "batch validations" for big files is coming.